packages/app-degree-pages/src/components/ListingPage/components/ProgramList/ListView/index.colums.config.js

```diff
@@ -117,10 +117,10 @@ const columns = [
     sortable: true,
     contentTemplate: ({ resolver, actionUrls }) => (
       <div>
-        {resolver.getConcurrentDegrees().lenght > 0 && (
+        {resolver.getConcurrentDegrees().then(data => data).length > 0 && (
           <div className="cell-container">concurrent</div>
         )}
-        {resolver.getAccelerateDegrees().length > 0 && (
+        {resolver.hasConcurrentOrAccelerateDegrees() > 0 && (
```

Comment:
Since `hasConcurrentOrAccelerateDegrees()` does a `!!` [on the length value and returns that](https://github.com/ASU/asu-unity-stack/blob/957cc815b0cbd70d6d90733dc9b2287720ad8434/packages/app-degree-pages/src/core/services/degree-data-prop-resolver-service.js#L85-L86), it would be better here to do a boolean rather than numerical comparison. It also makes more semantic sense, and improves readability since this is a "has" method.